### ISSUE TITLE
Replace raco submodule with pip install

### ISSUE THREAD

**Issue by bmyerz** [uwescience/myria-web#301]
@senderista and I discussed removing raco as a submodule. The reason is 1) difficulty developing raco (replace submodule with symlink or keep changing submodule commit hash for raco...) and 2) that it would be nice to point myria-web deployment at releases of raco.

Workflow for deploying/developing myria-web __without__ Raco hacking:
- pip install raco before running myria-web

Workflow for developing myria-web with Raco hacking:
- cd raco; python setup.py develop

**Comment by domoritz** [comment]
The problem was app engine, right?

**Comment by senderista** [comment]
Yes, it's been a while since I looked at this, but I think GAE restrictions on third-party libraries were the original reason for including raco as a submodule. We might be able to install it via `pip` using a `lib` directory under the GAE root directory: https://cloud.google.com/appengine/docs/python/tools/libraries27?hl=en#vendoring. Per the instructions on this page, we would add the following to `appengine_config.py`:
```
from google.appengine.ext import vendor

# Add any libraries installed in the "lib" folder.
vendor.add('lib')
```
Then we would add the desired raco version to `requirements.txt` and run `pip install -t lib -r requirements.txt` before deploying.
